testapps/stresstests/build.gradle

```diff
@@ -0,0 +1,80 @@
+apply plugin: 'com.android.application'
+
+def msalVersion = "2.+"
+
+if (project.hasProperty("distMsalVersion")) {
+    msalVersion = distMsalVersion
+}
+
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+    defaultConfig {
+        multiDexEnabled true
+        applicationId "com.msft.identity.client.sample"
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 1
+        versionName "1.0"
+    }
+
+    signingConfigs {
+        debug {
+            storeFile file('./debug.keystore')
+            storePassword 'android'
+            keyAlias 'androiddebugkey'
+            keyPassword 'android'
+        }
+
+        release {
```

Comment:
Do we need multiple configs for this project? We don't expect to release this I assume?

Reply:
You're right, no need for the release config.